pika/heartbeat.py

```diff
@@ -7,38 +7,38 @@
 
 
 class HeartbeatChecker(object):
-    """Checks to make sure that our heartbeat is received at the expected
-    intervals.
+    """Sends heartbeats to the broker and checks to make sure that the broker's
+    heartbeat is received before the expected timeout expires.
 
     """
-    DEFAULT_INTERVAL = 60
-    MAX_IDLE_COUNT = 2
     _CONNECTION_FORCED = 320
+    _MAX_IDLE_COUNT = 2
     _STALE_CONNECTION = "Too Many Missed Heartbeats, No reply in %i seconds"
 
-    def __init__(self, connection, interval=DEFAULT_INTERVAL, idle_count=MAX_IDLE_COUNT):
-        """Create a heartbeat on connection sending a heartbeat frame every
-        interval seconds.
+    def __init__(self, connection, interval):
```

Comment:
Since the doc string says "timeout" and not "interval" and interval is a half of the timeout value per spec (2 missed heartbeats are considered to be a timeout), should we rename the argument and properties?

Reply:
@vitaly-krugl

Reply:
I will update the documentation. We are sending heartbeats at a specified interval (half of what's passed in) so I think that's an appropriate name for it.

Reply:
@michaelklishin updated and pushed, let me know what you think. I already have a link to that mailing list thread in a comment as well.